### DISCUSSION TITLE
Import/Export settings

### DISCUSSION THREAD

**Discussion by Colehillkd** [RealRaven2000/QuickFolders#566] [Announcements]
My annual licence allows me to install onto more than one machine using the same email address.
I currently have it installed it on my desktop and have a few folders and other rules defined.
I would like to install onto my notebook, for use outside of the office, but I can't see a way of exporting my settings from the desktop so that I can have a similar setup on the notebook/laptop.  Is there an import/export option?
If not, then I think it would be a good idea.

**Comment by RealRaven2000** [comment]
The export functionality for QuickFolders settings is on the licenses page (restricted to Pro and Standard Users). It always exports all settings including the tabs, but not debug settings. When you import you get a choice of what to import:

![image](https://github.com/RealRaven2000/QuickFolders/assets/6314674/c91eb3d4-429f-4eac-9158-b59ec2a88f19)

To get there, open QuickFolders Options from the tool menu, then click on licenses. there will be a section "Backup/Restore" with what you need:

![image](https://github.com/RealRaven2000/QuickFolders/assets/6314674/46c24cd1-ce68-4888-8c1b-ca5abb2da6d3)
